configs/base_config.yaml

```diff
@@ -6,6 +6,7 @@ disable_wandb: false  # Disable WandB online integration
 # General
 seed: 0
 model: MLP
+metric: accuracy # accuracy or llm_tasks
```

Comment:
What's this for? Could you make the name more specific and add a comment about what it's supposed to do?